db/migrate/20190228210310_service_templates_should_have_names.rb

```diff
@@ -0,0 +1,13 @@
+class ServiceTemplatesShouldHaveNames < ActiveRecord::Migration[5.0]
+  class ServiceTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    ServiceTemplate.where(:name => [nil, ""]).each do |template|
+      say_with_time("Checking #{template} for name presence") do
+        template.update_attributes!(:name => "ServiceTemplate")
```

Comment:
If the name doesn't need to be unique and they'll all get the save value, then prefer `.update_all` rather than `update_attributes` on each instance.